cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
the root-level /cypress folder was probably created by running tests in the root directory instead of the client directory. We should get rid of these and keep all the cypress tests in /client.